app/controllers/staff/speakers_controller.rb

```diff
@@ -1,36 +1,26 @@
 class Staff::SpeakersController < Staff::ApplicationController
   decorates_assigned :speaker
-  before_filter :set_proposal, only: [:new, :create, :destroy]
+  before_filter :set_program_session, only: [:new, :create, :destroy]
 
   def index
-    render locals: {
-             proposals: @event.proposals.includes(speakers: :user).decorate
-           }
+    @program_session_speakers = Speaker.with_program_session.for_event(current_event)
```

Comment:
@Marty I tried to change this to current_event.speakers.with_program_session but it that goes through the proposals table to get speakers. As per a discussion with Zac I'm going to leave this for now, as I think it would take awhile to fix.